VERSION

```diff
@@ -1 +1 @@
-8.10.0.0
+8.10.0.0.pre.dev
```

Comment:
the `pre.dev` comes from running `make release`. Feel free to bump this to 8.10.0.1, release this change (for ent-search) and then bump to 8.10.0.2 for `main` (SNAPSHOT) is inferred.

Reply:
done